src/test/java/com/gildedrose/GildedRoseTest.java

```diff
@@ -19,4 +19,31 @@ void fooTest() {
         assertEquals(40, app.items[0].quality);
     }
 
+
+   @Test
+   void agedBrieTest() {
```

Comment:
typically I like test names that are explicit of what is being tested. and alternative can be the @DisplayName tag provided by Junit (preferably both).

When reading the tests or when the tests fails in the logs, seeing something like (aged brie decreases in quality every day) is a better indicator of what the test is doing

Reply:
Done